### PR TITLE
Fix rxmongo: Journal replay OutOfMemoryException

### PR DESCRIPTION
**Pull request by marcuslinke**
@scullxbones Fixes issue #111. Please review. Thanks!

### CODE REVIEW

File: rxmongo/src/test/scala/akka/contrib/persistence/mongodb/RxMongoJournallerSpec.scala [hunk 1]
```diff
@@ -41,10 +41,6 @@ class RxMongoJournallerSpec extends TestKit(ActorSystem("unit-test")) with RxMon
     val (range, head) = await(inserted)
     range should have size 1
 
-    underTest.journalRange("unit-test",1,3) onFailure {
```
Comment (marcuslinke) [inline rxmongo/src/test/scala/akka/contrib/persistence/mongodb/RxMongoJournallerSpec.scala]: I removed this. Maybe you have a better idea?

**Comment by marcuslinke** [comment]
@scullxbones Failed tests seems unrelated to my changes. Could you confirm please?

**Comment by scullxbones** [comment]
I think you're right.  I restarted the builds for mongo 3 and 3.2 that failed.

**Comment by scullxbones** [comment]
Looks good to me.  Thank you for the contribution!

**Pull request merged**